Tasks/AzureResourceGroupDeploymentV2/operations/ResourceGroup.ts

```diff
@@ -170,8 +170,24 @@ export class ResourceGroup {
             tl.error(error.message);
             if (error.details) {
                 tl.error(tl.loc("Details"));
+                var isPolicyVoilated = false;
                 for (var i = 0; i < error.details.length; i++) {
-                    var errorMessage = util.format("%s: %s %s", error.details[i].code, error.details[i].message, error.details[i].details);
+                    var errorMessage = null;
+                    if (error.details[i].code === "RequestDisallowedByPolicy") {
+                        if (!isPolicyVoilated) {
+                            let managementGroupId = this.getmanagementGroupId(error.details[i]);
+                            let portalUrl = this.taskParameters.endpointPortalUrl ? this.taskParameters.endpointPortalUrl : "https://ms.portal.azure.com";
+                            let policyLink = portalUrl + "#blade/Microsoft_Azure_Policy/PolicyMenuBlade/Assignments/scope/%2Fproviders%2FMicrosoft.Management%2FmanagementGroups%2F" + managementGroupId;
```

Comment:
Is there any other way to point to this?
You seem to be using a path to a particular blade in the portal, if the extension owner decides to change the path, the logged URLs will stop working.

Reply:
Discussed this with @imalokagrawal, I suppose this is fine for now. 
Should the word `scope` in the URL be replaced with the actual scope? i.e. of resource group or subscription?